packages/bpk-component-map/readme.md

```diff
@@ -70,12 +71,39 @@ export default () => (
 
 ### BpkMap
 
-| Property	      | PropType	| Required                	| Default Value |
-| --------------- | --------- | ------------------------- | ------------- |
-| mapRef          | func      | false                     | null          |
+| Property	       | PropType                                     | Required                 | Default Value |
+| ---------------- | -------------------------------------------- | ------------------------ | ------------- |
+| containerElement | node                                         | true                     | -             |
+| mapElement       | node                                         | true                     | -             |
+| region           | shape({latitude: number, longitude: number}) | true                     | -             |
+| mapRef           | func                                         | false                    | null          |
+| onRegionChange   | func                                         | false                    | null          |
+| onZoom           | func                                         | false                    | null          |
+| scrollEnabled    | bool                                         | false                    | true          |
+| zoom             | number                                       | false                    | 15            |
+| zoomEnabled      | bool                                         | false                    | true          |
+
+When using `withScriptjs`, some additional props are required:
+
+| Property	       | PropType                                     | Required                 | Default Value |
+| ---------------- | -------------------------------------------- | ------------------------ | ------------- |
+| googleMapURL     | string                                       | true                     | -             |
+| loadingElement   | node                                         | true                     | -             |
 
-When using `withScriptjs`, some additional props are required. Refer to [`withScriptjs` from `react-google-maps`](https://tomchentw.github.io/react-google-maps/#withscriptjs).
+### BpkOverlayView
 
-Refer to [`GoogleMap` from `react-google-maps`](https://tomchentw.github.io/react-google-maps/#withgooglemap) for all other props.
+| Property	       | PropType                                     | Required                 | Default Value      |
+| ---------------- | -------------------------------------------- | ------------------------ | ------------------ |
+| children         | node                                         | true                     | -                  |
+| position         | shape({latitude: number, longitude: number}) | true                     | -                  |
+| mapPaneName      | oneOf(MAP_PANE_NAMES)                        | false                    | overlayMouseTarget |
+
+
+## Methods
```

Comment:
Remove these (values can be retrieved using callbacks) and add alternative prop for `fitBounds`